rotkehlchen/tests/external_apis/test_cryptocompare.py

```diff
@@ -325,3 +326,23 @@ def test_cryptocompare_query_with_api_key(cryptocompare):
         match_main_currency=False,
     )
     assert price is not None
+
+
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+def test_starknet_historical_price_after_ticker_change(cryptocompare):
+    """Check that Starknet token price query after Cryptocompare ticker change is accurate.
+
+    It checks that price queries work properly after the May 9, 2024, switch
+    from STARK to STRK ticker. Regression test for https://github.com/rotki/rotki/issues/8892
+    """
+    a_strk = CryptoAsset.initialize(
+        identifier='STRK',
+        cryptocompare='STRK',
+        asset_type=AssetType.OWN_CHAIN,
+    )
```

Comment:
This will create an asset. Why not do `CryptoAsset(´STRK´)`. That would fetch it from the globaldb. I believe is better because it helps to detect errors from the data